delivery_roulier/tests/test_delivery_roulier.py

```diff
@@ -111,6 +108,7 @@ def test_roulier(self):
             self.assertEqual("get_label", roulier_args[1])
             roulier_payload = roulier_args[2]
             self.assertEqual(len(roulier_payload["parcels"]), 1)
+            # TODO: understand where ``weight`` is changed
```

Comment:
I'm afraid I cannot help you on this (I'm not familiar with this repo), but maybe @florian-dacosta could :crossed_fingers: ?